src/gui/editors/PianoRoll.cpp

```diff
@@ -3488,17 +3553,15 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 
 
 // responsible for moving/resizing scrollbars after window-resizing
-void PianoRoll::resizeEvent(QResizeEvent * re)
+void PianoRoll::resizeEvent(QResizeEvent* re)
 {
-	m_leftRightScroll->setGeometry( WHITE_KEY_WIDTH,
-								      height() -
-								SCROLLBAR_SIZE,
-					width()-WHITE_KEY_WIDTH,
-							SCROLLBAR_SIZE );
+	m_leftRightScroll->setGeometry(WHITE_KEY_WIDTH,
+								   height() - SCROLLBAR_SIZE,
+								   width() - WHITE_KEY_WIDTH,
+								   SCROLLBAR_SIZE);
```

Comment:
This just makes it look prettier and won't blow up with variable tab sizes.
```suggestion
	m_leftRightScroll->setGeometry(
		WHITE_KEY_WIDTH,
		height() - SCROLLBAR_SIZE,
		width() - WHITE_KEY_WIDTH,
		SCROLLBAR_SIZE
	);
```

Reply:
Sure, why not. I have no strong opinion either way. :) If it was up to me, I would probably keep it all on one long line. :D